src/__experimental__/components/label/label.spec.js

```diff
@@ -235,6 +235,14 @@ describe('Label', () => {
       expect(icon.exists()).toEqual(false);
     });
   });
+
+  describe.each(validationTypes)('when %s prop is passed as string and label is inline', (vType) => {
+    it('tooltip should render in position "top"', () => {
+      const wrapper = render({ [vType]: 'Message', useValidationIcon: true, inline: true }, mount);
+      const icon = wrapper.find(ValidationIcon);
+      expect(icon.prop('tooltipPosition')).toBe('top');
```

Comment:
As there is no test in this suite that checks for `tooltipPosition: right` this suite would pass if the tooltip incorrectly was always displayed on top.

We should add one that asserts when `inline:false` that `tooltipPosition: right`.

Reply:
Updated to include negative test case